hack/generate_client.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-GV="network:v1alpha1 servicemesh:v1alpha2 tenant:v1alpha1 devops:v1alpha1"
+GV="network:v1alpha1 servicemesh:v1alpha2 tenant:v1alpha1 devops:v1alpha1 volumesnapshot:v1beta1"
```

Comment:
this script will generate clientset for domain `kubesphere.io`, since your API objects is under domain `k8s.io`, I think it's gonna be a problem.